Zenodo.js

```diff
@@ -37,41 +37,43 @@
 
 function detectWeb(doc, url) {
 	if (url.indexOf('/record/')>-1) {
-		var collections = ZU.xpath(doc, '//div[contains(@class, "metadata")]//a[@class="navtrail"]');
+		var collections = ZU.xpath(doc, '//div[@itemscope]//span[@class="pull-right"]/span[contains(@class, "label-default")]');;
 		for (var i=0; i<collections.length; i++) {
-			//Z.debug(collections[i].textContent);
-			switch (collections[i].textContent) {
-				case "Software":
+			var type = collections[i].textContent.toLowerCase();
+			//Z.debug(type)
+			switch (type) {
+				case "software":
 					return "computerProgram";
-				case "Videos/Audio":
+				case "video/audio":
 					return "videoRecording";//or audioRecording?
-				case "Images":
+				case "figure":
+				case "drawing":
+				case "photo":
+				case "diagram":
+				case "plot":
 					return "artwork";
-				case "Presentations":
-				case "Posters":
+				case "presentation":
+				case "conference paper":
+				case "poster":
+				case "lesson":
 					return "presentation";
-				case "Lessons":
-				case "Books":
+				case "book":
 					return "book";
-				case "book-section":
-					return "chapter";
-				case "conference-papers":
-					return "conferencePaper"
-				case "patents":
+				case "book section":
+					return "bookSection";
+				case "patent":
 					return "patent";
-				case "reports":
-				case "working-papers":
-				case "technical-notes":
-				case "project-deliverables":
-				case "project-milestones":
-				case "proposals":
+				case "report":
+				case "working paper":
+				case "project deliverables":
+				case "preprint":
 					return "report";
-				case "Theses":
+				case "thesis":
 					return "thesis";
-				case "Datasets":
+				case "dataset":
 					//change when dataset as itemtype is available
-				case "journal-articles":
-				case "preprints":
+					return "document";
```

Comment:
so, I'm not sure about this: since we are using Extra anyway, there is no strong reason to use journal article anymore, is there? And in that case, using Document makes sense because it's rarely used otherwise -- the main reason being that it doesn't have its own CSL item type, but since we're changing the item type via Extra, that's not a problem. What do you think?